### DISCUSSION TITLE
Request to add to readme

### DISCUSSION THREAD

**Discussion by n0tf0und** [mvdwetering/huesyncbox#54] [General]
I am new to Home Assistant and sometimes confused by all of the different parts.
I wanted to add The Philips Hue Play HDMI Sync Box Integration.
Your directions start with "From HACS"
Problem was I had no idea what HACS was.
My suggestion:
Add a link to getting started with HACS
And maybe add on your Github> Installation
HACS (Home Assistant Community Store) 

To get the newbies started.
Seems obvious now, but was not as I explored the options to install.

BTW
Works great.
Thanks

**Comment by mvdwetering** [comment]
That is a valid point. I extended the HACS section in the readme.

**Comment by n0tf0und** [comment]
That will help a lot people who may not understand all of the ways to use HAThanks. . . It Can Wait . . .Safely sent from a mobile deviceOn Feb 25, 2023, at 11:21 AM, Michel van de Wetering ***@***.***> wrote:﻿
That is a valid point. I extended the HACS section in the readme.

**Comment by mvdwetering** [comment]
Cleaning up discussions. Closing this one as it is handled
